.github/workflows/ci.yaml

```diff
@@ -1,5 +1,9 @@
 name: Tests
-on: [pull_request, push]
+on:
+  push:
+    branches:
+      - main
+  pull_request:
```

Comment:
@marceloneppel if you just update the PR, it still going to trigger CI?

Reply:
It'll @paulomach. So we can update the PR with more commits and get feedback from CI again.

Reply:
Great. I believe this, as the PR template, should also be included everywhere! (at least I will do it ;)